CodeStarterCamp_Week2/main.swift

```diff
@@ -8,4 +8,24 @@
 
 import Foundation
 
-print("Hello, World!")
+var lottoNum: Set<Int> = []
+func lottoWin() {
+    while lottoNum.count < 6 {
+        lottoNum.insert(Int.random(in: 1...45))
+    }
+}
+
+lottoWin()
+
+let myLottoNumbers: [Int] = [1, 2, 3, 4, 5, 6]
+
+func drawLot() {
+    let num: Set<Int> = lottoNum.intersection(myLottoNumbers)
+    let numString = num.sorted().map {String($0)}.joined(separator: ", ")
```

Comment:
```suggestion
    let numString = num.sorted().map { String($0) }.joined(separator: ", ")
```
컨벤션은 정답은 없지만 보통 공백을 넣어주는 편입니다!